packages/neutron-std/src/serde/mod.rs

```diff
@@ -1,17 +1,70 @@
+use serde::{
+    de::{
+        self,
+        value::{I64Deserializer, U64Deserializer},
+    },
+    Deserialize,
+};
+use std::{
+    fmt::{Display, Formatter},
+    marker::PhantomData,
+    str::FromStr,
+};
+
+struct StringOrNumberVisitor<T> {
```

Comment:
Would be nice to have a description for this structure and what does it do

Reply:
done